articles/storage/storage-python-how-to-use-table-storage.md

```diff
@@ -181,7 +181,7 @@ table_service.delete_table('tasktable')
 Now that you've learned the basics of Table storage, follow these links
 to learn more.
 
-* [Python Developer Center](/develop/python/)
+* [Python Developer Center](https://azure.microsoft.com/en-us/develop/python/)
```

Comment:
@asears Hi Andrew, thanks for the fix on this! Very minor change needed on this, though--just remove the /en-us from the URL, as such:

`[Python Developer Center](https://azure.microsoft.com/develop/python/)`

This allows browsers running in locales other than en-us to redirect correctly to their localized version.

Thanks!